includes/discovery/discovery-protocols.inc.php

```diff
@@ -238,9 +237,9 @@
     echo PHP_EOL;
 } else {
     echo ' LLDP-MIB: ';
-    $lldp_array = snmpwalk_group($device, 'lldpRemTable', 'LLDP-MIB', 3);
+    $lldp_array = SnmpQuery::allowUnordered()->hideMib()->walk('LLDP-MIB::lldpRemTable')->table(3);
```

Comment:
Can we limit allowUnordered to a specific OS?  It is preferred since that option can cause inifinite loops.
(there is a field to set specific OIDs to be allowed to be unordered per-os)

Reply:
Makes sense. Just found an example of this.

Reply:
Solved.